examples/pt2/torch_compile_openvino/stable_diffusion/model_config.yaml

```diff
@@ -0,0 +1,19 @@
+minWorkers: 1
+maxWorkers: 1
+maxBatchDelay: 200
+responseTimeout: 3600
+deviceType: "cpu"
```

Comment:
Are we testing with CPU or GPU?

Reply:
If we add gpu to the device type, it's defaulting to nvidia execution. By keeping this to cpu and setting the device in openvino options to GPU, OpenVINO will internally run the model on GPU

Reply:
@suryasid This sounds like a bug in TorchServe. Expected behavior is to default to GPU if no config is specified or deviceType: "gpu"
Please file an issue for the same